Python/Product/PythonTools/PythonTools/Editor/PythonTextBufferInfo.cs

```diff
@@ -23,63 +27,90 @@
 using Microsoft.PythonTools.Repl;
 using Microsoft.VisualStudio.InteractiveWindow;
 using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
 using Microsoft.VisualStudio.Utilities;
 
 namespace Microsoft.PythonTools.Editor {
-    sealed class PythonTextBufferInfo : IDisposable {
+    sealed class PythonTextBufferInfo {
         public static PythonTextBufferInfo ForBuffer(PythonEditorServices services, ITextBuffer buffer) {
-            return buffer.Properties.GetOrCreateSingletonProperty(
+            var bi = buffer.Properties.GetOrCreateSingletonProperty(
```

Comment:
>buffer [](start = 21, length = 6)

The TryGetForBuffer checks for null param, so this one should probably throw ANE rather than NRE.

Reply:
👍